### PR TITLE
Docs: Fix inconsistencies for adding a site

### PR DESCRIPTION
**Pull request by GaryJones**

### CODE REVIEW

**Comment by GaryJones** [comment]
I was just trying to fix what the faux table of contents suggested - at least more descriptive than "main" folder?

(As someone new to VVV2, "main" folder means nothing to me.)

**Comment by cfoellmann** [comment]
for me the

> and The Main Folder

makes no sense at all. Would remove the part and reduce the headline to `vvv-custom.yml`

File: docs/en-US/adding-a-new-site/index.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ I'm going to walk through setting up a blog named vvvtest.com locally using VVV,
 
 If you're migrating a site from VVV 1, read this page, then visit the [migration page](migrating-vvv1.md) for further details.
 
-## `vvv-custom.yml` and The Main Folder
+## `vvv-custom.yml` and the Sites Folder
```
Comment (tomjn) [inline docs/en-US/adding-a-new-site/index.md]: This suggests a folder named `sites`, I'd prefer instead we used the root folder or removed it

Comment (GaryJones) [inline docs/en-US/adding-a-new-site/index.md]: Which do you prefer? 

1. `vvv-custom.yml` and the Root Folder
2. `vvv-custom.yml`

Comment (LoreleiAurora) [inline docs/en-US/adding-a-new-site/index.md]: #1

**Pull request merged**